.ci/static.sh

```diff
@@ -18,11 +18,15 @@ if [[ "$COMMAND" == "install" ]]; then
     conda install jupyter
     pip install codespell flake8 pylint
 elif [[ "$COMMAND" == "run" ]]; then
+    # Convert notebooks to Python scripts
     jupyter-nbconvert \
         --log-level WARN \
         --to python \
         --TemplateExporter.exclude_input_prompt=True \
         -- **/*.ipynb
+    # Remove style issues introduced in the conversion:
+    #   s/# $/#/g replaces lines with just '# ' with '#'
+    #   /get_ipython()/d removes lines containing 'get_ipython()'
```

Comment:
Doesn't warrant more comments in this file, but `get_ipython` raises flake8 errors because IPython injects it into the local namespace rather than importing it within the script, which honestly would make a lot more sense. In any case, `get_ipython` is only used for magic functions, which we wouldn't want to add to the script versions of these notebooks anyway.